### PR TITLE
Add Fire<> alias

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

File: styles/fire.h [hunk 1]
```diff
@@ -152,5 +152,12 @@ StyleAllocator StyleFirePtr() {
   return &factory;
 }
 
+// Base Fire, no Clash, Lockup or Off state
```
Comment (profezzorn) [inline styles/fire.h:155]: Need better explanation/comment.

Comment (fdarosa2663) [reply styles/fire.h]: Updated

File: styles/fire.h [hunk 2]
```diff
@@ -152,5 +152,12 @@ StyleAllocator StyleFirePtr() {
   return &factory;
 }
 
+// Simplified Fire, negates Clash, Lockup or Off FireConfig<> to use Fire with other Layered effects
+template<class COLOR1, class COLOR2, int DELAY=0, int SPEED=2, int BASE=0, int RAND=2000, int COOLING=5> 
+using Fire = StyleFire<COLOR1, COLOR2, DELAY, SPEED,
```
Comment (profezzorn) [inline styles/fire.h:157]: Just "Fire" is too generic a name.
Maybe StaticFire or Simplefire?

Comment (fdarosa2663) [reply styles/fire.h]: StaticFire works, updated

**Pull request merged**